runtime/interpreter/decode.go

```diff
@@ -1673,6 +1682,9 @@ func (d LocationDecoder) decodeAddressLocation() (common.Location, error) {
 	// Address
 
 	// Decode address at array index encodedAddressLocationAddressFieldKey
+	//
+	// TODO: Use `decodeAddressBytes` and remove the `checkEncodedAddressLength` below
```

Comment:
Can this already be done in this PR?

Reply:
I thought about it, but then it will introduce some inconsistency - i.e: locations will be metered during decoding, but not when created in-line.